index.js

```diff
@@ -54,7 +59,7 @@ exports.hookSystemJS = function(loader, exclude, coverageGlobal) {
       if (load.address.substr(0, System.baseURL.length) != System.baseURL)
         return source;
 
-      var name = load.address.substr(System.baseURL.length);
+      var name = path.normalize(load.address.substr(System.baseURL.length));
```

Comment:
Ok so it seems like this will ensure that `name` is then a system-formatted path style.